### PR TITLE
Remove Jacques Chester from CHARTER.md

### PR DESCRIPTION
**Pull request by jchester**
As I will no longer be working in a job focused on repository matters, I am resigning from the working group. Because I will not be active my name should be removed from voting members.

### CODE REVIEW

**Comment by jchester** [comment]
Paging @bobcallaway @brianf @Eh2406 @di @simi @jswank @MylesBorins @trevrosen @znewman01

A majority of you will need to approve this PR so that my name can be removed from the Maintainers list.

**Review by bobcallaway — APPROVED** [review]

**Review by di — APPROVED** [review]

**Review by simi — APPROVED** [review]
Sad to hear :(. Thanks for all the work done in this group.

**Review by znewman01 — APPROVED** [review]

**Comment by di** [comment]
We need six total votes for this to reach a majority (@jchester IMO your vote still counts until this is merged, and I'll take your authorship of this PR as a vote), which means we need one more approver from amongst the following list: @brianf @Eh2406 @jswank @MylesBorins @trevrosen

**Pull request merged**
